source/clear-linux/reference/compatible-kernels.rst

```diff
@@ -10,46 +10,50 @@ and provides links to their source code.
 Kernel native
 =============
 
-The *kernel-native* bundle focuses on the bare metal platforms. It is
-optimized for fast booting and performs best on the Intel® architectures
-described on the :ref:`supported hardware list<system-requirements>`. The
-optimization patches are found in our `Linux`_ GitHub\* repo.
+The *kernel-native* bundle focuses on the bare metal platforms. It is optimized
+for fast booting and performs best on the Intel® architectures described on the
+:ref:`supported hardware list<system-requirements>`.
+
+The optimization patches are found in the `Linux`_ GitHub\* repo.
 
 Kernel Container
 ================
 
-The *kernel-container* bundle contains the kernel used by the
-Intel® Clear Containers project. This kernel is optimized for
-fast booting and performs best on |CC| running on the Intel® architectures
-described on the :ref:`supported hardware list<system-requirements>`.
-The optimization patches are found in our `Linux-Container`_ GitHub repo.
+The *kernel-container* bundle contains the kernel used by the Intel® Clear
```

Comment:
Please remove this section. @DnPlas removed this bundle, `kernel-container`, on 09-06-2018. And Clear Containers are deprecated now.

Reply:
@mvincerx good one. Updated per your request.